Sources/Tracker/Utilities/TrackerConstants.swift

```diff
@@ -46,10 +47,16 @@ enum FailureReason: String, Codable, CaseIterable {
     
     case networkUnavailable = "network_unavailable"
     case lowBattery = "low_battery"
+    case socket_not_open = "socket_not_open"
+    
+    case token_not_found = "token_not_found"
```

Comment:
token_not_found, configs_not_found, feature_disabled are tracked when CS is not initialised and can be moved to Example app's constant